machines/Makefile

```diff
@@ -42,6 +41,7 @@ define set_machine_variables
 	$(eval VCS_DEFINES += +define+BSG_MACHINE_MAX_EPA_WIDTH=${BSG_MACHINE_MAX_EPA_WIDTH})
 	$(eval VCS_DEFINES += +define+BSG_MACHINE_MEM_CFG=${BSG_MACHINE_MEM_CFG})
 	$(eval VCS_DEFINES += +define+BSG_MACHINE_BRANCH_TRACE_EN=${BSG_MACHINE_BRANCH_TRACE_EN})
+	$(eval VCS_DEFINES += +define+den2048Mb+sg5+x16+FULL_MEM)
```

Comment:
What does this do? Is this something I need to add in F1? What does it do?

Reply:
we don't need that in bsg_f1. this is for the lpddr4 simulation model. If we are including mobile_ddr.v to the filelist, these macros have to be defined, otherwise the tool will crash.

https://github.com/bespoke-silicon-group/basejump_stl/blob/master/testing/bsg_dmc/lpddr_verilog_model/mobile_ddr.v#L109

Reply:
Can you add a comment describing this in the makefile here?

Reply:
Thanks!